@here/harp-datasource-protocol/lib/DecodedTile.ts

```diff
@@ -257,6 +257,8 @@ export interface PoiGeometry {
     technique?: number;
     stringCatalog?: Array<string | undefined>;
     objInfos?: AttributeMap[];
+    // Angle in degrees from north clockwise specifying the direction the icon can be shifted.
+    offsetDirection?: number;
```

Comment:
This should be a number array. Pois within a tile may have different offset directions.